cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -29,9 +29,16 @@ roles:
   - name: default
     type: system_role
     description: User exists, but have no permissions
-  - name: status_reporter
+  - name: manager_status_reporter
     type: system_role
-    description: User that can submit status reports
+    description: User that can submit status reports to manager nodes
```

Comment:
=>A user that can submit status reports for manager nodes
also for the other ones

Reply:
Comment above.